src/components/nav.js

```diff
@@ -1,21 +1,26 @@
 import React from 'react';
 import { NavLink } from 'react-router-dom';
+import { CgProfile } from 'react-icons/cg';
+import './styles/nav.css';
 
 const Nav = () => (
-  <nav>
-    <h1>Bookstore CMS</h1>
-    <ul>
-      <li>
-        <NavLink to="/">
+  <nav className="nav-container">
+    <h1 className="nav-heading">Bookstore CMS</h1>
+    <ul className="nav-ul">
+      <li className="nav-li">
+        <NavLink to="/" className="links">
```

Comment:
- [ ] Well done you did a very nice job but there are some little changes so kindly fix it.
- [ ] If you check the design the selected link is with this `#121212` and the link which is not selected is with this color `#fafafa` but in you can you just add one color so what you do when the use clicks on the book link the link color should be `#121212` and the other link color should be `#fafafa` same with categories link so kindly fix this issue. Check below the comparison between your design and Zaplin's design

**Your Design**
![Screenshot from 2023-07-20 19-11-41](https://github.com/zuhebahmed88091/Bookstore/assets/45226848/0557bcea-f666-4e17-a5e2-b2b9be55ae9f)

**Zaplin Design**
![Screenshot from 2023-07-20 19-11-41](https://github.com/zuhebahmed88091/Bookstore/assets/45226848/6bb7b300-bdca-4505-b6df-3f695b1e2861)

- [ ] Also if you check your Zaplin design all letters of link are capital letter so kindly add capital letter as well

Reply:
I have applied `useState ` and `useLocation` Hooks to resolve this issue.

- [x] Resolved.